webknossos/webknossos/dataset/dataset.py

```diff
@@ -1147,6 +1153,13 @@ def add_layer_from_images(
                 num_channels=pims_images.num_channels,
                 **add_layer_kwargs,  # type: ignore[arg-type]
             )
+            if (
+                pims_images.expected_shape.z == 1
+                and layer.data_format == DataFormat.Zarr
+            ):
+                if chunk_shape is None:
+                    chunk_shape = DEFAULT_CHUNK_SHAPE.with_z(1)
+                # chunks_per_shard is 1 by default for zarr atm
```

Comment:
I don't understand the comment in combination with the code, as the code only acts on the z dimension, but the chunks_per_shard requirement is probably for all dimensions?

Reply:
The code is about `chunk_shape`, and the comment about `chunks_per_shard`, which does not need to be set explicitly atm. However, I'll just add a similar code for chunks_per_shard, since it's future-proof for changes to the default.